source

```diff
@@ -96566,15 +96578,15 @@ new PaymentRequest(&hellip;); // Allowed to use
   <h4>Destroying documents</h4>
 
   <p id="discard-a-document">To <dfn data-x="destroy a document">destroy</dfn> a
-  <code>Document</code> <var>document</var>:</p>
+  <code>Document</code> <var>document</var> given an optional string <var>reason</var>:</p>
 
   <ol>
    <li><p><span data-x="destroy a document">Destroy</span> the <span
    data-x="nav-document">active documents</span> of each of <var>document</var>'s <span>descendant
    navigables</span>. <span class="XXX" data-x="">In what order?</span></p></li>
 
-   <li><p>Set <var>document</var>'s <i data-x="concept-document-salvageable">salvageable</i> state
-   to false.</p></li>
+   <li><p><span>Make document unsalvageable</span> given <var>document</var> and <var>reason
```

Comment:
The reason is optional, so we can't directly used it. Maybe say if the reason is not set, use "document destroyed"
 or something?

Also, nothing is currently setting the reason at this point. Are you going to add a caller to destroy with a reason set? There might be some future usage of the reason when @lozy219's https://github.com/whatwg/html/pull/8972 lands, but that will probably happen after your HTML PRs land. So maybe we can either remove the optional reason now (and just use "document-destroyed" directly here) or add a use for the reason in this PR.